### ISSUE TITLE
Change Themes folder

### ISSUE THREAD

**Issue by karneaud** [caffeinated/themes#84]
Is there any way to change the themes directory?

**Comment by tpaksu** [comment]
It's hardcoded here, if you can fork and change this line, I thinki it'll work:

https://github.com/caffeinated/themes/blob/c13045bd42f5a0753b3ed804f6c5100fe899f474/src/Theme.php#L59

**Comment by tpaksu** [comment]
And also fix symlinks:

https://github.com/caffeinated/themes/blob/c13045bd42f5a0753b3ed804f6c5100fe899f474/src/Theme.php#L129-L140

Maybe there are more. I suggest opening a PR for adding this path to the config file.

**Comment by kaidesu** [comment]
Yeah, currently its hardcoded. Would be open to a PR to add this in as a customizable option :+1:
